big_tests/tests/rest_helper.erl

```diff
@@ -501,6 +503,19 @@ make_malformed_msg_stanza_with_props(ToJID,MsgID) ->
             </properties>
         </message>">>).
 
+make_msg_stanza_with_thread(ToJID, MsgID, ThreadID, ThreadParentID) ->
```

Comment:
More correct name for this function is `make_msg_stanza_with_thread_and_parent`, but also need to have with and without `parent` attribute test case.